meson.build

```diff
@@ -183,12 +183,14 @@ features = []
 # Directories
 #-------------------------------------------------------------------------------
 
+prefixdir = get_option('prefix')
+
 sysconfdir = get_option('sysconfdir')
 cdata.set_quoted('SYSCONFDIR', sysconfdir)
 
-bindir = join_paths(get_option('prefix'), get_option('bindir'))
-includedir = join_paths(get_option('prefix'), get_option('includedir'))
-libdir = join_paths(get_option('prefix'), get_option('libdir'))
+bindir = join_paths(prefixdir, get_option('bindir'))
+includedir = join_paths(prefixdir, get_option('includedir'))
+libdir = join_paths(prefixdir, get_option('libdir'))
```

Comment:
I'm thinking that maybe we should move all the xxxxdir = ... get_option() here and into a single block. Something like below:

```
prefixdir = get_option('prefix')
sysconfdir = get_option('sysconfdir')
bindir = join_paths(prefixdir, get_option('bindir'))
includedir = join_paths(prefixdir, get_option('includedir'))
libdir = join_paths(prefixdir, get_option('libdir'))
datadir =  .... ?
bashcompletiondir =  ...
fishcompletiondir =  ...
zshcompletiondir = ...

```

Because it seems we forgot to also add prefixdir to datadir.  Then you use the variables above in the arrays to denote the default value from user, if set.

Reply:
Not going to lie, the way meson deals with paths is quite annoying IMHO. In particular - if the user has provided (implicitly or explicitly) relative one, the prefix will be added when working with meson constructs but otherwise you have to add it.

</rant>

Sure can do. I will also take the opportunity to sort the summary, based on your example.